### PR TITLE
Video 6759 results mobile styling

### PR DESCRIPTION
**Pull request by olipyskoty**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- [VIDEO-6759](https://issues.corp.twilio.com/browse/VIDEO-6759)

### Description

This PR adds mobile designs for the Results page. Instead of making the pane scrollable, I made some of the padding and margins smaller, though I am open to other suggestions.

**iPad Air Screenshots**:
IMG_0017

IMG_29114B0640F4-1

### Before review
* [ ] Updated CHANGELOG.md if necessary
* [x] Added unit tests if necessary
* [ ] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [x] Included screenshot as PR comment (if needed)
* [x] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Re-tested if necessary

### CODE REVIEW

File: src/components/panes/Results/Results.tsx [hunk 1]
```diff
@@ -8,28 +8,68 @@ import { SmallError } from '../../../icons/SmallError';
...
-    '&:not(:last-child)': {
-      paddingBottom: '2em',
-      borderBottom: '0.1em solid #8891AA',
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    header: {
+      float: 'left',
+      [theme.breakpoints.down('md')]: {
+        float: 'initial',
+      },
```
Comment (timmydoza) [inline src/components/panes/Results/Results.tsx:17]: Here I wonder if some `padding-bottom` would help to give a bit more room between the buttons and the results. It's subtle.

```
      [theme.breakpoints.down('md')]: {
        float: 'initial',
        paddingBottom: '1em'
      },
``` 

Before:
image

After:
image

**Review by timmydoza — CHANGES_REQUESTED** [review]
Looks great! Nice implementation!

Just wanted to also mention the potential bug we talked about, where the user should not be able to proceed past this screen when there is an error:
image

Not sure why this is happening. Does it correctly stop the user when there is a real error? In my case, my token server has expired, so if this works right for real errors then maybe this issue can be solved when we add an error for an expired server. 

Also, I was poking around, and I think we also need to apply mobile styles to the Snackbar as well:
image

**Comment by olipyskoty** [comment]
> Looks great! Nice implementation!
> 
> Just wanted to also mention the potential bug we talked about, where the user should not be able to proceed past this screen when there is an error: 
> 
> Not sure why this is happening. Does it correctly stop the user when there is a real error? In my case, my token server has expired, so if this works right for real errors then maybe this issue can be solved when we add an error for an expired server.
> 
> Also, I was poking around, and I think we also need to apply mobile styles to the Snackbar as well:

Looks like we were correctly disabling the down arrow button when the preflight test failed, but we weren't hiding the following pane. We were not accounting for the `preflight-token-failed` at all however. So I added this to the logic for disabling the down button, and hiding the following pane. I think it will be good to include this behavior even though the user may not even be able to make it past the "Get Started" pane once we implement the expired server error handling.

**Review by timmydoza — APPROVED** [review]
Looks great - nice work! 

I think the Snackbar component could use an error that explains when it is shown to the user, but other than that I think this is good to go!

**Pull request merged**